dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -78,193 +70,76 @@
-
-  .callout2
-    %table
-      %tr
-        %td Use this callout?
-        %td Callout name (localization_key)
-        %td Callout text
-        %td Element callout refers to (element_id)
-        %td Location of qtip on callout
-        %td Location on element qtip points to
-        %td Qtip x distance adjust
-        %td Qtip y distance adjust
-        %td Hide qtip
-      %tr
-        %td
-          %input{:class => "use_callout_2", :type => 'checkbox'}
-        %td
-          -#stretch goal: disable this input and autogenerate callout name (e.g. frozen_cross_rotate_callout0...callout1, etc.)
-          %input{:class => "localization_key_2", :type => 'text', :placeholder => 'Name of callout'}
-        %td
-          %input{:class => "callout_text_2", :type => 'text', :placeholder => 'Text of callout'}
-        %td
-          -#stretch goal: change this input to a select and list all of the elements and sub-elements of the particular level (e.g. [block-id='1'], #runButton, #turn_right_qtip > .blocklyEditableText:eq(1), etc.)
-          %input{:class => "element_id_2", :type => 'text', :placeholder => 'ID of element'}
-        %td
-          %select{:class => "qtip_position_my_2"}
-            %option{:value => 'bottom left'} Bottom left
-            %option{:value => 'bottom center'} Bottom center
-            %option{:value => 'bottom right'} Bottom right
-            %option{:value => 'right bottom'} Right bottom
-            %option{:value => 'right center'} Right center
-            %option{:value => 'right top'} Right top
-            %option{:value => 'top right'} Top right
-            %option{:value => 'top center'} Top center
-            %option{:value => 'top left'} Top left
-            %option{:value => 'left top'} Left top
-            %option{:value => 'left center'} Left center
-            %option{:value => 'left bottom'} Left bottom
-        %td
-          %select{:class => "qtip_position_at_2"}
-            %option{:value => 'top right'} Top right
-            %option{:value => 'top center'} Top center
-            %option{:value => 'top left'} Top left
-            %option{:value => 'left top'} Left top
-            %option{:value => 'left center'} Left center
-            %option{:value => 'left bottom'} Left bottom
-            %option{:value => 'bottom left'} Bottom left
-            %option{:value => 'bottom center'} Bottom center
-            %option{:value => 'bottom right'} Bottom right
-            %option{:value => 'right bottom'} Right bottom
-            %option{:value => 'right center'} Right center
-            %option{:value => 'right top'} Right top
-        %td
-          %input{:class => "qtip_position_adjust_x_2", :type => 'number'}
-        %td
-          %input{:class => "qtip_position_adjust_y_2", :type => 'number'}
-        %td
-          %input{:class => "qtip_hide_2", :type => 'checkbox'}
+  - (0...3).each do |x|
+    %label Callout #{x+1}
+
+    %div{:class => "callouts#{x}"}
```

Comment:
Note this doesn't block check in at all, and if you prefer I can fix this up from RubyMine (1 click to update all occurrences in the file).